Sources/TokamakCore/Tokens/Font.swift

```diff
@@ -340,17 +430,30 @@ public struct _FontProxy {
   }
 }
 
-struct FontKey: EnvironmentKey {
-  static let defaultValue: Font? = nil
+enum FontPathKey: EnvironmentKey {
+  static let defaultValue: [Font] = []
 }
 
 public extension EnvironmentValues {
+  var _fontPath: [Font] {
+    get {
+      self[FontPathKey.self]
+    }
+    set {
+      self[FontPathKey.self] = newValue
+    }
+  }
+
   var font: Font? {
     get {
-      self[FontKey.self]
+      _fontPath.first
     }
     set {
-      self[FontKey.self] = newValue
+      if let newFont = newValue {
+        _fontPath = [newFont] + _fontPath.filter { $0 != newFont }
+      } else {
+        _fontPath = []
+      }
     }
```

Comment:
Could this file be split by any chance to avoid the linter warning? Maybe some extensions or types moved to separate files?

Reply:
Sure, I'll split it into more manageable files.